sql/core/src/test/scala/org/apache/spark/sql/execution/DataSourceScanExecRedactionSuite.scala

```diff
@@ -31,7 +31,7 @@ import org.apache.spark.sql.test.SharedSQLContext
 abstract class DataSourceScanRedactionTest extends QueryTest with SharedSQLContext {
 
   override protected def sparkConf: SparkConf = super.sparkConf
-    .set("spark.redaction.string.regex", "file:/[\\w-_@/]+")
+    .set("spark.redaction.string.regex", "file:/[^\\]\\s]+")
```

Comment:
Actually matching `[^\\]]+` should be enough. Matching spaces also for double insurance.